src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/Application.kt

```diff
@@ -0,0 +1,29 @@
+package no.brreg.informasjonsforvaltning.organizationcatalogue
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import no.brreg.informasjonsforvaltning.organizationcatalogue.configuration.AppProperties
+import org.springframework.scheduling.annotation.EnableScheduling
+import org.springframework.data.mongodb.core.mapping.event.ValidatingMongoEventListener
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean
+import org.springframework.boot.SpringApplication
+import org.springframework.context.annotation.Bean
+
+@SpringBootApplication
+@EnableConfigurationProperties(AppProperties::class)
+@EnableScheduling
+open class Application {
+    @Bean
+    open fun validatingMongoEventListener(): ValidatingMongoEventListener {
+        return ValidatingMongoEventListener(validator())
+    }
+
+    @Bean
+    open fun validator(): LocalValidatorFactoryBean {
+        return LocalValidatorFactoryBean()
+    }
```

Comment:
Er disse beans i bruk noen steder?

Reply:
Nå er det over ett år siden jeg faktisk la de til, så er ikke helt sikker på at dette er korrekt. Men etter som jeg husker så måtte jeg legge de til for å kunne bruke `@NotBlank` valideringen i OrganizationDB